scheduler/test/Scheduler.test.ts

```diff
@@ -89,7 +89,7 @@ describe("Schedule", () => {
       console.log("task_id:" + decode_log.args.task_id);
       await schedule.cancelCall(ethers.utils.hexlify(decode_log.args.task_id));
     } else {
-      expect(false).to.not.be.ok;
+      expect(false).to.be.ok;
```

Comment:
is there anything like `fail` we can use?

Reply:
```
expect.fail("custom error message");
```